From 0985e89a11c8696fc1537b54516216c362676ec9 Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Thu, 5 Jun 2014 12:33:31 -0400
Subject: [PATCH 67/72] [ovirt] elide passwords in logcollector.conf

This configuration file may contain passwords of the form:

And:

Add these to the existing 'passwd_files' processing added in
commit 3da5697.

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/ovirt.py | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sos/plugins/ovirt.py b/sos/plugins/ovirt.py
index 969440b..acd6ad3 100644
--- a/sos/plugins/ovirt.py
+++ b/sos/plugins/ovirt.py
@@ -121,6 +121,7 @@ class Ovirt(Plugin, RedHatPlugin):
         )
 
         passwd_files = [
+            "logcollector.conf",
             "imageuploader.conf",
             "isouploader.conf"
         ]
@@ -131,6 +132,11 @@ class Ovirt(Plugin, RedHatPlugin):
                 r"passwd=(.*)",
                 r"passwd=********"
             )
+            self.do_file_sub(
+                conf_path,
+                r"pg-pass=(.*)",
+                r"pg-pass=********"
+            )
 
         sensitive_keys = self.DEFAULT_SENSITIVE_KEYS
         #Handle --alloptions case which set this to True.
-- 
1.9.3

